packages/terra-search-field/tests/jest/SearchField.test.jsx

```diff
@@ -46,7 +46,7 @@ describe('Snapshots', () => {
   });
 
   it('renders a search field with an aria-label', () => {
-    const searchField = shallowWithIntl(<SearchField inputAttributes={{ 'aria-label': 'Search Field' }} />).dive();
+    const searchField = shallowWithIntl(<SearchField inputAttributes={{ ariaLabel: 'Search Field' }} />).dive();
```

Comment:
We'll want this to still work with `inputAttributes={{ 'aria-label': 'Search Field'}}`, otherwise this becomes a breaking change. Consumers may be using this syntax to set custom aria-labels.

We'll also now need to support `inputAttributes={{ 'ariaLabel': 'Search Field'}}`